app/views/layouts/design_system.html.erb

```diff
@@ -5,17 +5,21 @@
     <meta name="govuk:components_gem_version" content="<%= GovukPublishingComponents::VERSION %>">
     <%= javascript_include_tag "admin/domain-config" %>
     <%= javascript_include_tag "govuk_publishing_components/load-analytics" %>
+    <%= javascript_include_tag "admin/modules/ga4-page-view-tracking" %>
```

Comment:
I don't think you need to do this - it should be included as part of application.js?